guide/resources/design-challenges.md

```diff
@@ -57,6 +57,10 @@ Bob has been gifted a $100 bitcoin gift card for his birthday and wants to redee
 
 Bob Jr wants to receive his weekly allowance in bitcoin. Design a low-fidelity app with flows that lets him share a [request]({{ '/guide/daily-spending-wallet/requesting/#sharing-the-request' | relative_url }}) for allowance with his parents, who may or may not have a bitcoin wallet.
 
+## Challenge 6: Private purchase
+
+Bob wants to buy an engagement ring to surprise Alice. He has to pay on-chain because it's a large expense. Alice recently sent him sats, so she has a view into the sats in Bob's wallet. Design a low-fidelity flow Bob might follow to spend his sats using [PayJoin](https://en.bitcoin.it/wiki/PayJoin). In contrast to a naive transaction, thanks to PayJoin, even if Alice sees the resulting transaction on-chain she won't be able to discern whether it is a small transfer, output consolidation, or the large purchase that would spoil the surprise.
```

Comment:
Question on terminology. Is this supposed to be a "naive transaction" or a "native transaction"?

Reply:
It's supposed to be "naïve", meaning natural or straightforward. I can add the diacritic to make it more explicit.